### DISCUSSION TITLE
Extract relevant feature, roll_time_series and LSTM

### DISCUSSION THREAD

**Discussion by heib6xinyu** [blue-yonder/tsfresh#1078] [Q&A]
Should the input of the function extract relevant feature be the entire time series frame (including X and y) and y, return feature (which can be directly used as X)?

**Comment by nils-braun** [comment]
Hi @heib6xinyu !
You can find an example of using the function in our documentation: https://tsfresh.readthedocs.io/en/latest/api/tsfresh.convenience.html#tsfresh.convenience.relevant_extraction.extract_relevant_features

As you see, you need to pass in the time series (does not include the y), separate from the y. The returned data will contain the selected features.
